shuffle/move/examples/e2e/message.test.ts

```diff
@@ -27,13 +27,56 @@ Deno.test("Ability to set message", async () => {
 });
 
 Deno.test("Ability to set NFTs", async () => {
+  // Initialize nft_collection resource for both sender and receiver
+  let senderInitializeTxn = await main.initializeNFTScriptFunction(
+    context.defaultUserContext,
+  );
+  senderInitializeTxn = await devapi.waitForTransactionCompletion(
+    senderInitializeTxn.hash,
+  );
+  const secondUserContext = context.UserContext.fromEnv("test");
+  let receiverInitializeTxn = await main.initializeNFTScriptFunction(
+    secondUserContext,
+  );
+  receiverInitializeTxn = await devapi.waitForTransactionCompletion(
+    receiverInitializeTxn.hash,
+  );
+  assert(senderInitializeTxn.success);
+  assert(receiverInitializeTxn.success);
+
+  // Mint TestNFT into sender address
   const contentUri = "https://placekitten.com/200/300";
-  let txn = await main.createTestNFTScriptFunction(contentUri);
+  let txn = await main.createTestNFTScriptFunction(
+    context.defaultUserContext,
```

Comment:
In a fast follow (FF), lets have this method default to defaultUserContext, and remove the param.